app/javascript/packages/address-search/components/full-address-search.spec.tsx

```diff
@@ -320,4 +320,57 @@ describe('FullAddressSearch', () => {
       await expect(handleLocationsFound).to.eventually.be.called();
     });
   });
+
+  context('Address Search with Results Section Heading', () => {
```

Comment:
What do you think about adding a test to ensure this section does not render if nothing is passed in. This is the behavior in IDP so think a test here would be a nice addition.